encrypted-media/scripts/playback-retrieve-persistent-usage-record.js

```diff
@@ -39,10 +39,11 @@ function runTest(config, testname) {
 
             assert_in_array(  event.messageType, [ 'license-request', 'individualization-request' ] );
 
-            config.messagehandler( config.keysystem, event.messageType, event.message ).then( function( response ) {
+            config.messagehandler( event.messageType, event.message ).then( function( response ) {
 
-                _mediaKeySession.update( response )
-                .catch(function(error) {
+                _mediaKeySession.update( response ).then(function() {
+                    _video.setMediaKeys(_mediaKeys);
```

Comment:
Yes. I'm about to push a PR with more comprehensive tests for different timing for setMediaKeys. We should choose one location for this call for all the other tests. I wanted to test this test on Edge and that seemed to need this call here.

Of course, I could also polyfill this problem with Edge, but that's petty messy.